LICENSE.md

```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
```

Comment:
We really didn't need this to be styled.

Reply:
Restored in 4d6f07a8cdee5a208770942c606f96a100f51402